src/lib/create_midpoint.js

```diff
@@ -1,27 +1,40 @@
 import * as Constants from '../constants';
 
+/**
+ * Returns GeoJSON for a Point representing the
+ * midpoint of another feature.
+ *
+ * @param {GeoJSON} parent
+ * @param {GeoJSON} startVertex
+ * @param {GeoJSON} endVertex
+ * @param {Object} map
+ * @return {GeoJSON} Point
+ */
 export default function(parent, startVertex, endVertex, map) {
   const startCoord = startVertex.geometry.coordinates;
   const endCoord = endVertex.geometry.coordinates;
 
   // If a coordinate exceeds the projection, we can't calculate a midpoint,
   // so run away
-  if (startCoord[1] > Constants.LAT_RENDERED_MAX ||
+  if (
+    startCoord[1] > Constants.LAT_RENDERED_MAX ||
     startCoord[1] < Constants.LAT_RENDERED_MIN ||
     endCoord[1] > Constants.LAT_RENDERED_MAX ||
-    endCoord[1] < Constants.LAT_RENDERED_MIN) {
+    endCoord[1] < Constants.LAT_RENDERED_MIN
+  ) {
     return null;
   }
 
-  const ptA = map.project([ startCoord[0], startCoord[1] ]);
-  const ptB = map.project([ endCoord[0], endCoord[1] ]);
-  const mid = map.unproject([ (ptA.x + ptB.x) / 2, (ptA.y + ptB.y) / 2 ]);
+  const ptA = map.project([startCoord[0], startCoord[1]]);
+  const ptB = map.project([endCoord[0], endCoord[1]]);
+  const mid = map.unproject([(ptA.x + ptB.x) / 2, (ptA.y + ptB.y) / 2]);
 
-  return {
+  const midpoint = {
     type: Constants.geojsonTypes.FEATURE,
     properties: {
+      ...parent.properties,
```

Comment:
It's worth noting that, per [MDN's browser compatibility docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax), IE does not support spread syntax.

Reply:
That's a good spot, I think a `for ... in` loop might cover it for IE support